src/reconcile/rec_write.c

```diff
@@ -40,9 +40,9 @@ typedef struct {
 
 	/*
 	 * Track the oldest running transaction and whether to skew lookaside
-	 * to the newest or oldest update.
+	 * to the newest update.
 	 */
-	bool las_skew_oldest;
```

Comment:
This may appear gratuitous, but I believe it fixes a real bug when checkpoint does eviction, where the resulting pages don't choose the oldest values but also don't choose the newest.  I made this change here because I thought this was involved in the `modify` issue.  lmk if it should be separated out.